projects/vehicles/protos/mercedes_benz/MercedesBenzSprinterSimple.proto

```diff
@@ -4,7 +4,6 @@
 # Simple kinematic model of the Mercedes-Benz Sprinter to be moved with a Supervisor.
 
 EXTERNPROTO "webots://projects/vehicles/protos/abstract/VehicleWheel.proto"
```

Comment:
I'm getting warnings about missing declarations on the simple version of the vehicle:

```
ERROR: Missing declaration for 'GlossyPaint', add: 'EXTERNPROTO "../../../webots_master/projects/appearances/protos/GlossyPaint.proto"' to '/home/daniel/webots_master/projects/vehicles/protos/mercedes_benz/MercedesBenzSprinterSimple.proto'.
ERROR: Missing declaration for 'RoughPolymer', add: 'EXTERNPROTO "../../../webots_master/projects/appearances/protos/RoughPolymer.proto"' to '/home/daniel/webots_master/projects/vehicles/protos/mercedes_benz/MercedesBenzSprinterSimple.proto'.
ERROR: Missing declaration for 'PerforatedMetal', add: 'EXTERNPROTO "../../../webots_master/projects/appearances/protos/PerforatedMetal.proto"' to '/home/daniel/webots_master/projects/vehicles/protos/mercedes_benz/MercedesBenzSprinterSimple.proto'.
```